ui-components/src/locales/general.json

```diff
@@ -1235,6 +1235,206 @@
     "WI": "Wisconsin",
     "WY": "Wyoming"
   },
+  "neighborhoods": {
```

Comment:
If we're not translating these, do we need this in the loacales json?

Reply:
Moved this to the building details form file.